cypress/support/elements/researcher_anonymous_classes_elements.js

```diff
@@ -136,5 +136,58 @@ export const ResearcherClassesElements = {
   },
   getSaveButton() {
     return cy.get('.action_menu_header_right input ').eq(0);
+  },
+
+  // Class Dashboard
+
+  verifyResearcherHeader() {
+    cy.get('#app [class^=header--researcherHeader--]').contains("Researcher View");
+  },
+  verifyAnonymizeStudentsToggleNotDisplayed() {
+    cy.get('[data-cy=anonymize-students]').should("not.exist");
+  },
+  verifyStudentNameInProgressDashboard() {
+    cy.get('[data-cy=student-list] [data-cy=student-name]').should("contain", "Student 37");
```

Comment:
Where does "Student 37" come from? (Here & below.) I don't see where this was set such that it is ready to be verified here. It does not seem to be in any of the files in the "helpers" or "testdata" directories either. Is this implicitly assuming that we will always use a certain set of data that is not specified? Or is it specified somewhere that I'm missing?

Reply:
@johnTcrawford I used the portal URL "/portal/offerings/335/external_report/" to automate this feature. In this report, there are four students assigned to it. In that one of the students is **Student 37**. Once we launched the report we saw this student 37 was the first student for this report. This not going to be changed since I created this data only for this automation purposes. Hence, I hardcoded this data to verify the student's name.